README.md

```diff
@@ -17,15 +17,11 @@ Note: search isn't functional in the development environment.
 
 ## Troubleshooting
 
-We heavily rely on caching for performance issues so if some changes are not reflected in the resulting portal try cleaning cache by running:
+If some changes are not reflected in the resulting portal try cleaning cache by running:
 
     yarn clean
```

Comment:
I think this is leftover from the gatsby generated README. I _think_ it is still relevant, but the "we heavily rely on caching" was confusing to me, because it refers to the gatsby team - not us, the owners of the repo.